src/sentry/utils/integrationdocs.py

```diff
@@ -111,6 +114,45 @@ def sync_docs():
     for platform_id, platform_data in iteritems(data['platforms']):
         for integration_id, integration in iteritems(platform_data):
             sync_integration_docs(platform_id, integration_id, integration['details'])
+            python_platforms.append(get_integration_id(platform_id, integration_id))
+
+    # create python platform constants
+    __sync_platform_types(python_platforms)
+
+
+def __sync_platform_types(platforms):
+    """
+    Updating existing platform types whenever docs are synced
+    """
+    # Cannot use six here
+    valid_platforms = set(platform for platform in platforms)
```

Comment:
There is no way to use six here. And unicode() is not allowed by the linter. What would be appropriate to ensure they are all consistent?

Reply:
@mattrobenolt